src/ngraph/node.hpp

```diff
@@ -554,6 +556,13 @@ namespace ngraph
         ///
         /// TODO: Make a plan to deprecate this.
         std::shared_ptr<NodeType> get_node_shared_ptr() const { return m_node; }
+        /// \return A useable shared pointer to this output. If index 0, the node,
+        /// otherwise find or create a GOE.
+        std::shared_ptr<Node> as_node_shared_ptr() const NGRAPH_DEPRECATED("Transitional.")
```

Comment:
How about `as_single_output_node`?

Reply:
BTW, I'm not sure we need to deprecate this necessarily. Even once GOE is gone, it still could make sense to have something that dangles out an `Identity` node hooked up to the _i_th input.